tools/isobuild/bundler.js

```diff
@@ -592,6 +593,15 @@ class File {
     this._contents = options.data || null; // contents, if known, as a Buffer
     this._hashOfContents = options.hash || null;
     this._hash = null;
```

Comment:
That would require bumping the manifest file version, which we might want to do.

Maybe then we can also fix https://github.com/meteor/meteor-feature-requests/issues/54 at the same time. :-)